src/s3fs.cpp

```diff
@@ -1940,6 +1951,11 @@ static int s3fs_utimens(const char* _path, const struct timespec ts[2])
 
     S3FS_PRN_INFO("[path=%s][mtime=%lld][ctime/atime=%lld]", path, static_cast<long long>(ts[1].tv_sec), static_cast<long long>(ts[0].tv_sec));
 
+    struct timeval now;
+    gettimeofday(&now, /*tz=*/ NULL);
```

Comment:
I think this should use `s3fs_clock_gettime` instead of `gettimeofday`.

Reply:
`s3fs_clock_gettime` is local to `cache.cpp` and `S3fsLog::GetCurrentTime` does not call it.  I don't understand the advantage of `s3fs_clock_gettime` in this situation?

Reply:
It was originally caused by POSIX.1-2008, I remember that clock_gettime is recommended over gettimeofday.
If you just want to get the current time like this process, you can use gettimeofday.
But I think it's better to use clock_gettime.
(Although it is different from this case, clock_gettime is faster (option dependent) and more accurate in time measurement etc.)
Also, if necessary, we can remove static from the declaration of s3fs_clock_gettime.